src/lib/components/collapsiblepanel/CollapsiblePanel.component.js

```diff
@@ -0,0 +1,114 @@
+//@flow
+import React from "react";
+import { lighten } from "polished";
+import styled, { css } from "styled-components";
+import Button from "../button/Button.component";
+import { mergeTheme } from "../../utils";
+import * as defaultTheme from "../../style/theme";
+
+type Props = {
+  expand?: Boolean,
+  onHeaderClick?: any => void,
+  onClick? : any => void,
+  children?: Array,
+  node?: String,
+  variant? : string,
+};
+
+const ARROW = {
+  colapsed: 'fas fa-angle-down',
+  expanded: 'fas fa-angle-up',
+}
+
+const PanelText = styled.div`
+`;
+
+const CollapsiblePanelContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  border-radius: 8px 8px 4px 4px;
+
+  ${props => {
+    const brandingTheme = mergeTheme(props.theme, defaultTheme);
+
+    return css`
+        background-color: ${brandingTheme[props.variant]};
+        color: ${brandingTheme.background};
+      `;
+    }
+  }
+`;
+
+
+const HeaderContainer = styled.div`
+  height: ${defaultTheme.navbarHeight};
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: ${defaultTheme.padding.small} ${defaultTheme.padding.large}
+`;
+
+const ExpandedItems = styled.div`
+  height: ${defaultTheme.navbarHeight};
+  display: flex;
+  align-items: center;
+  padding: ${defaultTheme.padding.smaller} ${defaultTheme.padding.large};
+
+  ${props => {
+    const brandingTheme = mergeTheme(props.theme, defaultTheme);
+    const brandLight = lighten(0.1,  brandingTheme[props.variant]).toString();
+    return props.onClick
+      && css`
+        &:hover {
+          cursor: pointer;
+          background-color: ${brandLight};
+        }
+      `;
+    }
+  }
+`;
+
+const ExpandedContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+function CollapsiblePanel({
+  expanded = false,
+  variant =  'primary',
+  onHeaderClick,
+  onClick,
+  children = [],
+  node = '',
+}: Props) {
+  return (
+    <CollapsiblePanelContainer className="sc-collapsiblepanel" variant={variant} >
+      <HeaderContainer>
+        <PanelText className="sc-chips-text">
+          {node}
+        </PanelText>
+        <Button
```

Comment:
Use `inverted` props ;)
```
 <Button
      icon={<i className={expanded ? ARROW.expanded : ARROW.colapsed} />}
      inverted
      size="larger"
    />
```

Reply:
When I use `inverted` my Icon gets the same colour as my background and it gets "invisible" only to appear on hover with a very light shade of gray. Those are the Button's default variant colours but they don't go well with my `background-color: defaultTheme.brand.primary ` , so I set this Button's `variant` as `base` because it gets a light shade of blue, from all the other `variants` I think this one is better but I don't know if it is the best approche here.

This is how it looks:

`inverted`

|` on hover default variant` |` variant set as base `|
|---------------------------------|-----------------------------|
| ![image](https://user-images.githubusercontent.com/31314523/69837572-3e39ea80-122e-11ea-994b-0f7775ed5c50.png) ![image](https://user-images.githubusercontent.com/31314523/69837280-b2738e80-122c-11ea-9662-828b97995b0d.png) | ![image](https://user-images.githubusercontent.com/31314523/69837225-6d4f5c80-122c-11ea-91ff-fe89dc9641ab.png)|

`!inverted`

| `on hover default variant` |
|---------------------------------|
| ![image](https://user-images.githubusercontent.com/31314523/69837655-9bce3700-122e-11ea-8b84-8bc2b7778ee8.png) ![image](https://user-images.githubusercontent.com/31314523/69837402-5c531b00-122d-11ea-9241-5dc9dd5b2318.png) |

Reply:
oh! I think we don't need this Button, only icons are enough since, we moved `onHeaderClick` to `HeaderContainer`. Sorry I missed it

Reply:
No problem! I'll fix it later